### PR TITLE
fix: css and html crate types + html minify

### PR DESCRIPTION
**Pull request by alexander-akait**

### CODE REVIEW

File: crates/html_node/src/lib.rs [hunk 1]
```diff
@@ -116,6 +116,7 @@ fn minify_inner(code: &str, opts: MinifyOptions) -> anyhow::Result<String> {
                 let mut gen = CodeGenerator::new(
                     &mut wr,
                     CodegenConfig {
+                        minify: true,
```
Comment (alexander-akait) [inline crates/html_node/src/lib.rs:119]: We forget to set it

File: packages/css/lib/index.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   content: Buffer,
   options: any
 ): Promise<binding.TransformOutput> {
-  return binding.minify(content, toBuffer(options));
+  return binding.minify(content, toBuffer(options ?? {}));
 }
 
 export function minifySync(content: Buffer, options: any) {
-  return binding.minifySync(content, toBuffer(options));
+  return binding.minifySync(content, toBuffer(options ?? {}));
```
Comment (alexander-akait) [inline packages/css/lib/index.ts:11]: Allow to avoid options for better DX (the same for other)

**Review by kdy1 — APPROVED** [review]
Thanks!

File: packages/css/lib/index.ts (hunk 2 above)
Comment (yisibl) [reply packages/css/lib/index.ts:11]: Would we be better off using `Option<String>` in Rust?

**Pull request merged**